Runtime/AvatarCreator/UI/Elements/AccountCreationElement.cs

```diff
@@ -24,27 +23,24 @@ private void OnEnable()
         {
             sendEmailButton.onClick.AddListener(OnSendEmailButton);
```

Comment:
In theory we could make this slightly more flexible by removing the auto assigning of the event listeners and instead make the appropriate functions public and assign them in the inspector. It would also mean that you might need the button references. But not a blocker for me if you think it's unnecessary to do that.